hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/OMDBInsightEndpoint.java

```diff
@@ -211,10 +254,53 @@ public Response getOpenKeyInfo(
         break;
       }
     }
+    // Populate the keysSummary map
+    createKeysSummaryForOpenKey(keysSummary);
+
+    openKeyInsightInfo.setKeysSummary(keysSummary);
+
     openKeyInsightInfo.setLastKey(lastKey);
```

Comment:
This seems incorrect, because lastKey will still be overridden from each other (FSO vs NonFSO). We should provide only 1 argument in API end point for type of buckets (FSO or NonFSO) at a time.